README.md

```diff
@@ -8,10 +8,10 @@ Image Name: `moderntribe/squareone-php`
 
 ### Supported Tags
 
-* `latest`: The latest Dockerfile version using the most recent PHP version (currently 7.4),
+* `latest`: The latest Dockerfile version using the most recent PHP version (currently 8.0),
   built from the `master` branch.
-* `74-latest`: The latest Dockerfile version for PHP 7.4, built from the `master` branch.
-* `74-<x.y.z>`: Specific Dockerfile versions for PHP 7.4, built based on matching tags.
```

Comment:
We're not actually removing 7.4, just adding 8.0, so we should leave 7.4 in the docs as a support tag.